lib/deep_cover/core_ext/coverage_replacement.rb

```diff
@@ -4,13 +4,32 @@
 
 module DeepCover
   module CoverageReplacement
+    OLD_COVERAGE_SENTINEL = Object.new
+    ALL_COVERAGES = {lines: true, branches: true, methods: true}.freeze
+
     class << self
       def running?
         DeepCover.running?
       end
 
-      def start
-        return if running?
+      def start(targets = OLD_COVERAGE_SENTINEL)
+        if targets == OLD_COVERAGE_SENTINEL
+          # Do nothing
+        elsif targets == :all
+          targets = ALL_COVERAGES
+        else
+          targets = targets.to_hash.select { |key, value| ALL_COVERAGES.include?(key) && value }
+          targets = Hash[targets.map { |k, v| [k, !!v] }]
```

Comment:
The ruby builtin uses `opt = rb_convert_type(opt, T_HASH, "Hash", "to_hash");`, i'm copying them

Reply:
Calling `to_hash` is great. I commented on the following line: instead of `Hash[x]`, use `x.to_h`. Not sure if there's a robocop rule for that, but there should be :-)

Reply:
Oh, lol.